testutil/helpers.go

```diff
@@ -28,14 +30,28 @@ func Unmount(t *testing.T, mountPoint string) {
 
 // RequiresRoot skips tests that require root, unless the test.root flag has
 // been set
-func RequiresRoot(t *testing.T) {
+func RequiresRoot(t testing.TB) {
 	if !rootEnabled {
 		t.Skip("skipping test that requires root")
 		return
 	}
 	assert.Equal(t, 0, os.Getuid(), "This test must be run as root.")
 }
 
+// RequiresRootM is similar to RequiresRoot but intended to be called from *testing.M
+// as in RequiresRootM(os.Stderr, os.Exit)
+func RequiresRootM(stderr io.Writer, osExit func(code int)) {
```

Comment:
Why are you taking stderr and the os.Exit functions to this?  Wouldn't it always be the same for testing?